src/lib.rs

```diff
@@ -168,6 +168,56 @@ impl Order {
         self.account.get(&mut self.nonce, challenge_url).await
     }
 
+    /// Deactivate the order's pending or valid authorizations
+    ///
+    /// Returns the updated status of the order post-deactivation.
+    ///
+    /// This is useful when you want to cancel a pending order you wish to abandon, or to revoke
+    /// valid authorizations for the order's identifiers to prevent reuse.
+    ///
+    /// Once deactivated the order and associated resources (authorizations, challenges, etc) can
+    /// not be updated further.
+    pub async fn deactivate(&mut self) -> Result<&OrderState, Error> {
+        #[derive(Serialize)]
+        struct DeactivateRequest<'a> {
+            status: &'a str,
```

Comment:
Would it make semantic sense to have this take an `AuthorizationStatus`?

Reply:
Fixed. I added a commit implementing serialize for `AuthorizationStatus` into a separate commit. I can fold that into the one adding deactivation if preferred.

Reply:
Yes, would prefer squashing.

Reply:
Squashed :foot: